frontend/public/components/namespace.jsx

```diff
@@ -94,7 +95,16 @@ const ProjectRow = ({obj: project}) => {
   </ResourceRow>;
 };
 
-export const ProjectList = props => <List {...props} Header={ProjectHeader} Row={ProjectRow} />;
+// TODO Should be using PatternFly empty-state pattern for things like this
```

Comment:
This `TODO`, is more or less what you are replicating here, correct?  Or are you thinking we need to push it further?

Reply:
I'm imitating an empty state here but I think it might need a little more thought. This is just the first of many adjustments that will need to be made in the case that a user logs in for the first time and there are no projects/namespaces available to them. This is one case where pf-react would come in handy because we could use the EmptyState component where needed.

Reply:
We can still use base Patternfly empty state styles if needed without the React component.

Console does have some existing empty state pages. I think in the short term it would be good to align with those for internal consistency.